src/renderer/shared/ui/Modals/BaseModal/BaseModal.tsx

```diff
@@ -61,7 +58,6 @@ const BaseModal = ({
                   name="close"
                   size={20}
                   className="absolute top-[18px] right-[14px] z-10"
-                  ariaLabel={t('basemodal.closeButton')}
```

Comment:
I guess we can live without it for a while 😄 
`const { t } = useI18n();` was wrong inside pure UI component